src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public class Task {
+    private String description;
+    private boolean marking;
+
+    public Task(String description) {
+        this.description = description;
+        marking = false;
+    }
+
+    public void markDone() {
```

Comment:
Perhaps it would be better to have a method to print the string, rather than having a function that does both setting the boolean and printing